nailgun/entities.py

```diff
@@ -6687,6 +6687,7 @@ def __init__(self, server_config=None, **kwargs):
             ),
             'product': entity_fields.OneToManyField(Product),
             'sync_date': entity_fields.DateTimeField(required=True),
+            'foreman_tasks_recurring_logic_id': entity_fields.IntegerField(),
```

Comment:
Is the recurring_logic a entity, If yes, please raise a issue to add the entity and replace the IntegerField by it, after the issue is resolved.

Reply:
@san7ket Recurring logic entity already exists in a nailgun code base here -> https://github.com/SatelliteQE/nailgun/blob/23d93a316197dd3089ec5e69a6ce08eea9b1dddb/nailgun/entities.py#L5254. If you want I can create the issue for this change to add entity_field in sync plan.

Reply:
`'foreman_tasks_recurring_logic_id': entity_fields.RecurringLogic(),`

Reply:
make sense now, earlier I did not understand your question. It is better to make it as you mentioned as above.

Reply:
Ignore the comment above, 

'foreman_tasks_recurring_logic_id':entity_fields.OneToOneField(RecurringLogic),

Reply:
Ok, Good to ACK then

Reply:
Why not change this now? Instead of raising another PR later (which I think would be a breaking change)

Reply:
@sean797 I'm holding this PR till the time I see the right way to implement this. I'm also thinking how to achieve this as foreman_tasks_recurring_logic_id is just a read-only field